datadog/provider.go

```diff
@@ -38,24 +44,22 @@ func Provider() terraform.ResourceProvider {
 }
 
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
-
-	config := Config{
```

Comment:
I've killed this feature that seemed overkill to me, the indirection level made things more complicated for nothing